project/Dependencies.scala

```diff
@@ -2,7 +2,7 @@ import sbt._
 
 object Dependencies {
 
-  val codacyPluginsVersion = "19.0.10_play_2.7"
+  val codacyPluginsVersion = "20.0.0-splittoolspart.8.ecb1486b_play_2.7"
```

Comment:
Just leaving here the reminder to replace this with a supported version 😄

Reply:
✅ Updated to the stable version `20.0.0`.